scss/mixins.scss

```diff
@@ -20,262 +17,106 @@ $_o-brand-default-not-set-warning: false !default; // Has the default not set wa
     @if type-of($brand) != 'string' or $brand == '' {
```

Comment:
I think we should split these error messages into more targeted error messages:
https://github.com/Financial-Times/o-brand/blob/f8911dc95e21ea11a7650e5747cb4257c3c64549/scss/mixins.scss#L14-L19

A specific error message for when the component is an empty string and another for when the brand is an empty string.